manifests/g/GovernmentOfThePRC/ShangHai/PuDong/ZhiHuiPuDong/3.8.0/GovernmentOfThePRC.ShangHai.PuDong.ZhiHuiPuDong.locale.zh-CN.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: GovernmentOfThePRC.ShangHai.PuDong.ZhiHuiPuDong
+PackageVersion: 3.8.0
+PackageLocale: zh-CN
+Publisher: 上海市浦东新区人民政府
+PublisherUrl: https://www.pudong.gov.cn/

```

Comment:
```suggestion
PublisherUrl: https://english.pudong.gov.cn/
# https://www.pudong.gov.cn/ - 中文页面在 winget-bot 的验证失败：PreconditionFailed
# 请见：https://github.com/microsoft/winget-pkgs/pull/199049#issuecomment-2543861326
```

![image](https://github.com/user-attachments/assets/f698efb5-a629-47d8-8d8b-f34e5f394cd4)

Can this be waived?